crates/sui-graphql-rpc/src/types/open_move_type.rs

```diff
@@ -12,18 +12,33 @@ pub(crate) struct OpenMoveType {
     signature: OpenMoveTypeSignature,
 }
 
+/// The basic four abilities for a move object
 #[derive(Enum, Copy, Clone, Eq, PartialEq)]
 pub(crate) enum MoveAbility {
+    /// Allows values of types with this ability to be copied.
     Copy,
+    /// Allows values of types with this ability to be popped/dropped.
     Drop,
+    /// Allows the type to serve as a key for global storage operations.
     Key,
+    /// Allows values of types with this ability to exist inside a struct in global storage.
     Store,
```

Comment:
I would also tend to prefer a shorter phrasing than "Allows value of types with this ability to...", how about:

```suggestion
    /// Enables values to be copied.
    Copy,
    /// Enables values to be popped/dropped.
    Drop,
    /// Enables values to be held directly in global storage.
    Key,
    /// Enables values to be held inside a struct in global storage.
    Store,
```